lib/jnpr/jsnapy/operator.py

```diff
@@ -2844,19 +2844,19 @@ def _create_message(text):
                                         re.search('-', del_val)):
                                     dvalue = abs(float(delta_val.strip('%')))
                                     mvalue = val1 - ((val1 * dvalue) / 100)
-                                    if (val2 > val1 or val2 < mvalue):
-                                        flag_pass = False
-                                    else:
+                                    if (val2 < val1 and val2 >= mvalue):
```

Comment:
this code block doesn't account for times when the pre and post value remains the same, such as interface down pre and post, and input/output bps being 0. I propose:
```
if (val2 < mvalue):
	flag_pass = False
else:
	flag_pass = True
```